art/attacks/evasion/fast_gradient.py

```diff
@@ -250,8 +250,8 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
 
     def _check_params(self) -> None:
         # Check if order of the norm is acceptable given current implementation
-        if self.norm not in [np.inf, int(1), int(2)]:
-            raise ValueError("Norm order must be either `np.inf`, 1, or 2.")
+        if self.norm not in ["inf", np.inf, int(1), int(2)]:
```

Comment:
Could you make this line to be consistent with in other files:  
if self.norm not in [1, 2, np.inf, "inf"]:

Reply:
Fixed.